README.md

```diff
@@ -268,7 +268,7 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `database.external.clairDatabase`                                           | The database used by clair                                                                                                                                                                                                                                                                                                                      | `clair`                         |
 | `database.external.notaryServerDatabase`                                    | The database used by Notary server                                                                                                                                                                                                                                                                                                              | `notary_server`                 |
 | `database.external.notarySignerDatabase`                                    | The database used by Notary signer                                                                                                                                                                                                                                                                                                              | `notary_signer`                 |
-| `database.external.sslmode` | Connection method of external database (require, prefer, disable) | `disable` |
+| `database.external.sslmode`                                                 | Connection method of external database (require, verify-full, verify-ca, disable)                                                                                                                                                                                                                                                               | `require` |
```

Comment:
The default value should be `disable` as we set it in `values.yaml` https://github.com/goharbor/harbor-helm/blob/master/values.yaml#L484.
And could you add the comment in the `values.yaml` to list the valid values for `sslmode` either?

Reply:
Good point. Description added to `values.yaml` and default fixed.

Reply:
The default value hasn't updated yet.

And could you squash all the commits into single one?